cp-quickstart/start-docker-cloud.sh

```diff
@@ -6,6 +6,10 @@
 #########################################
 
 NAME=`basename "$0"`
+QUIET="${QUIET:-true}"
+[ -z $QUIET ] && 
```

Comment:
@rspurgeon not sure if I'm interpreting this correctly, but given the line

```
QUIET="${QUIET:-true}"
```

Is it the case that 

1. `QUIET` will never have a non-zero length? (i.e., `-z` will always fail)
2. Due to above, `REDIRECT_TO` will always go to `/dev/null`?

Can you please elaborate on the desired behavior?

Reply:
Yes, that was confusing.  I think my original intent was for the user to turn off "QUIET" with QUIET="" but that doesn't seem correct.  I changed it in latest push to be if QUIET=="true" otherwise...